TypeCobol/Compiler/CodeElements/Expressions/StorageArea.cs

```diff
@@ -67,6 +67,24 @@ public override string ToString()
             return base.ToString();
         }
 
+        public override bool Equals(object obj)
+        {
+            return Equals(obj as StorageArea);
+        }
+
+        public bool Equals(StorageArea storageArea)
+        {
+            return System.Object.ReferenceEquals(this, storageArea);
+        }
+
+        public override int GetHashCode()
+        {
+            unchecked
+            {
+                return (13 * 397) ^ base.GetHashCode();
+            }
+        }
+
```

Comment:
This is useless. This is equivalent to use the default implementations from the framework. We should discuss if StorageArea actually needs a custom equality comparison.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1600/commits/8b7686fa9f02c00e2a7150589c65d7fd66b787c8